frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx

```diff
@@ -355,15 +354,22 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
 
           <div className="page-body">
             <div className="container-xl">
+              {isLoadingGroup && (
+                <div className="row">
+                  <div className="skeleton-line w-10 col mx-3"></div>
+                </div>
+              )}
```

Comment:
```suggestion
              {isLoadingGroup ? <Loader /> : (<div className="card">//Rest of the components// </div>)}
```
Shall we re-use the Loader `src/ManageSSO/Loader.jsx`

Reply:
done